src/systems/eu/components/ecl-navigation-eu/ecl-navigation-side-eu/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@ec-europa/ecl-navigation-side-eu",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "0.0.1",
+  "description": "ECL Side Navigation Menu",
+  "main": "ecl-navigation-side-eu.scss",
+  "module": "ecl-navigation-side-eu.js",
+  "style": "ecl-navigation-side-eu.scss",
+  "dependencies": {
+    "@ec-europa/ecl-base-eu": "^0.0.1",
+    "@ec-europa/ecl-expandables-eu": "^0.0.1",
+    "@ec-europa/ecl-grid-eu": "^0.0.1",
```

Comment:
Following the connection in terms of grids:
- `@ec-europa/ecl-grid` depends on `@ec-europa/ecl-base-ec` because it's our internal QA rule, however the implementation doesn't seem to actually utilize any variables from the base.
- In my eyes, this makes the `@ec-europa/ecl-grid` a low-handing fruit to simply move to `generic` and use directly in sub-systems

To be discussed with the team